### PR TITLE
lab 1

### PR DESCRIPTION
**Pull request by sevenzing**
> image

> image

### CODE REVIEW

**Comment by ntdesmond** [comment]
> > image
> 
> > image

i have more hehehe

UPD: meet the guest of honour
image

File: app_python/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "currenTTime"
```
Comment (Ilya-Kolomin) [inline app_python/pyproject.toml]: Small typo :)

**Review by Ilya-Kolomin — APPROVED** [review]
Review OK👍

File: app_python/PYTHON.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Best practices for Python web development
+
++ for simple web application, use simple web framework - **Flask**. It's simple, and syncrhonous fast-to-write
++ use wsgi for production, not flask dev server
++ use docker for your application for easy production run
++ use ready .gitignore file from [here](https://github.com/github/gitignore/blob/main/Python.gitignore)
++ use EVN variables to configure your app without rebuilding
```
Comment (bragov4ik) [inline app_python/PYTHON.md]:
```suggestion
+ use ENV variables to configure your app without rebuilding
```

**Review by bragov4ik — CHANGES_REQUESTED** [review]
LGTM besides a typo

**Review by bragov4ik — APPROVED** [review]
LGTM

**Review by pminina01 — APPROVED** [review]
Good

**Review by Anatolay — APPROVED** [review]

**Pull request closed without merging**